src/screens/SendSheet.js

```diff
@@ -882,7 +889,12 @@ export default function SendSheet(props) {
     const currentProviderNetwork = ethereumUtils.getNetworkFromChainId(
       Number(currentProvider._network.chainId)
     );
-    const assetNetwork = isL2Asset(selected?.type) ? selected.type : network;
+    const assetNetwork = isNft
+      ? selected.network
+      : isL2Asset(selected?.type)
+      ? selected.type
+      : network;
```

Comment:
Please refactor this nested ternary 🙏 
It would be best to make it a helper function like `getAssetNetwork` can be local to the file I think.

Reply:
shit you caught me